source/shared/net/desktop-wallet/transaction-log-filter.rst

```diff
@@ -0,0 +1,31 @@
+.. _transaction-log:
+
+==============================
+Apply a transaction log filter
+==============================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Specify which transactions to show in the transaction log
+=========================================================
+
+#. Go to **Accounts** and select the account that you want to apply the transaction log filter to.
+
+#. Select **Change view**.
+
+#. Select **Transaction log**.
+
+#. Select **log filters**.
+
+#. Specify the date range that you want the log filter to cover
+
+#. Select the transaction types you want displayed in the log.
+
+#. Select **Apply and save**.
```

Comment:
Something had gone wrong in the build here. It is supposed to be three buttons that says "Search", "Save filters" and "Clear". 

- "Search" will apply the filters momentarily so people can use the function to look up specific transactions 
- "Save filters" will save the filters untill they are cleared, or new ones are saved.
- "Clear" will clear the filters.

Reply:
The build I have now says Apply, Save, and Clear

Reply:
Yeah, same here. It turns out that the Search, Save filters and Clear wording had accidentally been merged to the wrong branch, so that's why we have Apply, Save and Clear at the moment. It will be updated before release.